components/ui/Textarea.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { cn } from '../../lib/utils';
+
+export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(({ className, ...props }, ref) => {
```

Comment:
Our `StyledTextarea`  component has a counter that makes the limit really clear. Any change we could have something similar here?